mmedit/datasets/pipelines/augmentation.py

```diff
@@ -1031,3 +1031,86 @@ def __repr__(self):
         repr_str = self.__class__.__name__
         repr_str += (f'(keys={self.keys})')
         return repr_str
+
+
+@PIPELINES.register_module()
+class CopyValueFromKey:
+    """Copy the value of a source key to a destination key.
+
+    It does the following: results[dst_key] = results[src_key].
+
+    Args:
+        src_key (str): The source key.
+        dst_key (str): The destination key.
+    """
+
+    def __init__(self, src_key, dst_key):
+
+        self.src_key = src_key
+        self.dst_key = dst_key
+
+    def __call__(self, results):
+        """Call function.
+
+        Args:
+            results (dict): A dict containing the necessary information and
+                data for augmentation.
+
+        Returns:
+            dict: A dict with a key added/modified.
+        """
+
+        results[self.dst_key] = results[self.src_key].copy()
+        return results
+
+    def __repr__(self):
+        repr_str = self.__class__.__name__
+        repr_str += (f'(src_key={self.src_key})')
+        repr_str += (f'(dst_key={self.dst_key})')
+        return repr_str
+
+
+@PIPELINES.register_module()
+class RoundClipZeroOne:
```

Comment:
So this is a kind of quantization?

Reply:
Yes it is. After the degradations, the low-quality images are not quantized. This function just turns them to quantized images

Reply:
Okay the name "round clip" is not very clear. It is not simply a clip after a round. How about naming it as "Quantize"?